package.xml

```diff
@@ -24,10 +16,9 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>roscpp</build_depend> 
-  <build_depend>rospy</build_depend> 
+  <depend>roscpp</depend>
+  <depend>rospy</depend>
```

Comment:
With the script gone, we can actually say goodbye to this dependency too ;)

Reply:
Rebased and removed rclpy dependency in 460092b